services/resourceGroups/handlers.go

```diff
@@ -0,0 +1,19 @@
+package resourceGroups
+
+import (
+	"encoding/json"
+	"net/http"
+)
+
+// PutResourceGroupsHandler is the PUT method handler for /subscriptions/{subscription-id}/resourceGroups
+func PutResourceGroupsHandler(subscriptionID string, resourceGroupName string) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		b, err := json.Marshal(NewSubscriptionResponse(subscriptionID, resourceGroupName))
+		if err != nil {
+			panic(err)
+		}
+		w.Header().Set("Content-Type", "application/json")
+		w.Header().Set("Charset", "UTF-8")
+		w.Write(b)
```

Comment:
I'm confused by this. Shouldn't a resourceGroups/handler have a NewResourceGroupResponse ?

Reply:
It should. The resource group stuff is mostly a dirty stub because I got sidetracked making the new subscription handler since I (for some reason) thought we needed it for creating resource groups.